lib/experimental/new_embed.dart

```diff
@@ -1167,26 +1103,20 @@ class ConsoleExpandController extends ConsoleController {
         footer = DElement(footer),
         expandIcon = DElement(expandIcon),
         _expanded = false,
-        super(consoleElement) {
-    super.console.setAttr('hidden');
+        super(DElement(consoleElement),
+            errorClass: 'text-red', filter: filterCloudUrls) {
+    super.element.setAttr('hidden');
     footer.removeAttribute('hidden');
     expandButton.onClick.listen((_) => _toggleExpanded());
   }
 
-  void appendError(String error) {
-    super.appendError(error);
+  void showOutput(String message, {bool error = false}) {
+    super.showOutput(message, error: error);
     if (!_expanded && error != null) {
```

Comment:
Does 1115-1117 still make sense given the change in what `error` means?

Reply:
nope, fixed.